spec/features/lender_contributes_to_loan_request_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.feature "lender contributes to loan request" do
+  let(:lender) { User.create(email: "example@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.